# Goal
string room_name

---
# Result
bool success
string final_message

---
# Feedback
string current_location
